content/en/docs/reference/config/analysis/ist0162/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: GatewayPortNotDefinedOnService
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+This message occurs when a gateway (usually `istio-ingressgateway`) offers a
+port that the Kubernetes service workload selected by the gateway does not.
+
+For example, your Istio configuration contains these values:
+
+{{< text yaml >}}
+# Gateway with bogus port
+
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: httpbin-gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*"
+  - port:
+      number: 8004
+      name: http2
+      protocol: HTTP
+    hosts:
+    - "*"
+---
+
+# Default Gateway Service
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-ingressgateway
+spec:
+  selector:
+    istio: ingressgateway
+  ports:
+  - name: status-port
+    port: 15021
+    protocol: TCP
+    targetPort: 15021
+  - name: http2
+    port: 80
+    protocol: TCP
+    targetPort: 8080
+  - name: https
+    port: 443
+    protocol: TCP
+    targetPort: 8443
+{{< /text >}}
+
+In this example, the `GatewayPortNotDefinedOnService` message occurs because this
+configuration uses port 8004, but a default `IngressGateway` is only open on target ports
```

Comment:
I think `IngressGateway` should be consistent with line 8, which uses a different name.